stable/verdaccio/templates/NOTES.txt

```diff
@@ -1,5 +1,9 @@
 1. Get the application URL by running these commands:
-{{- if contains "NodePort" .Values.service.type }}
+{{- if .Values.ingress.enabled }}
+{{- range .Values.ingress.hosts }}
+  http://{{ . }}
```

Comment:
Depending on `tls` config, should be `https`.

Reply:
happy to change this, but I can't find a good example on how to match the `ingress.hosts` array with the `ingress.tls[].hosts` to determine `http` vs `https`. Mind giving me a hand with the syntax?

Reply:
not super esthetic but that work:
```
{{- $tls := .Values.ingress.tls }}
{{- if .Values.ingress.enabled }}
{{- range $host := .Values.ingress.hosts }}
{{- if $tls }}
https://{{ $host }}
{{- else }}
http://{{ $host }}
{{- end }}
{{- end }}
```